### ISSUE TITLE
Apidocs - Telescope, go back

### ISSUE THREAD

**Issue by MathieuDR** [emmanueltouzery/elixir-extras.nvim#6]
Imagine you select the wrong module, Is there an easy way to go back to the previous selection?

**Comment by emmanueltouzery** [comment]
Yeah.. I could make a shortcut in the second telescope picker, like control-something, to do that. It makes sense, I'll probably do it!
